frontend/src/app/services/game.ts

```diff
@@ -88,7 +91,11 @@ export class Game {
     const nextDate = nextDay(lastDate);
     this.moveForwardMitigations();
     const mitigationEffect = this.calcMitigationEffect(nextDate);
-    const dayState = this.simulation.simOneDay(mitigationEffect, this.rng.getRandomness());
+    let realHistory;
+    if (this.simulation.lastDate < this.scenario.dates.rampUpEndDate) {
```

Comment:
Otherwise it is undefined?
Consider use conditional operator:
```
const realHistory = (this.simulation.lastDate < this.scenario.dates.rampUpEndDate) ? this.scenario.realRampUpHistory : undefined;
```

Reply:
Done